### PR TITLE
Big cleanup and allow other hosts to be used besides localhost

### PR DESCRIPTION
**Pull request by solarkennedy**
In preparation to allow people to send sensu events to yocalhost from inside docker (once that is portforwarded/synapse/whatever), I would like to make the library flexible enough to do that.

Also adding flake8

cc @mrtyler @bobtfish 

Internal ticket OPS-4459

### CODE REVIEW

File: pysensu_yelp/__init__.py [hunk 1]
```diff
@@ -146,7 +165,7 @@ def send_event(name, runbook, status, output, team, page=False, tip=None, notifi
     json_hash = json.dumps(result_dict)
     sock = socket.socket()
     try:
-        sock.connect(SENSU_ON_LOCALHOST)
+        sock.connect(sensu_host, sensu_port)
```
Comment (somic) [inline pysensu_yelp/__init__.py]: Will this work? I think socket.connect() takes one arg - (addr, port) tuple, not 2 separate args.

File: tox.ini [hunk 2]
```diff
@@ -1,10 +1,18 @@
 [tox]
 envlist = py26,py27,p34
 
+[flake8]
+# TODO: fix long lines
+max-line-length = 2000
```
Comment (mrtyler) [inline tox.ini]: lol wut

File: pysensu_yelp/__init__.py (hunk 1 above)
Comment (mrtyler) [inline pysensu_yelp/__init__.py]: good catch afaik

**Comment by mrtyler** [comment]
fix'n'ship

File: pysensu_yelp/__init__.py (hunk 1 above)
Comment (solarkennedy) [inline pysensu_yelp/__init__.py]: Whoa! Thanks!

**Pull request merged**